### PR TITLE
message: fix msg_received and seq_diff check-ups

### PR DESCRIPTION
**Pull request by freeHackOfJeff**

### CODE REVIEW

File: modules/tools/diagnostics/message.py [hunk 1]
```diff
@@ -93,9 +93,9 @@ def callback(self, data):
             time = 0
             sequence_num = 0
 
-        if self.msg_received == True:
+        if self.msg_received:
             seq_diff = sequence_num - self.sequence_num
-            if seq_diff is not 0:
+            if seq_diff == 0:
```
Comment (kechxu) [inline modules/tools/diagnostics/message.py]: It is not correct.

Comment (freeHackOfJeff) [reply modules/tools/diagnostics/message.py]: Um..., thanks for your review!  That's my mistake, will fix it ASAP.

**Comment by Capri2014** [comment]
@freeHackOfJeff It is there in case diagnosis tool receive a topic with no increase  seq number.

**Comment by Capri2014** [comment]
@freeHackOfJeff Your changes to result2pb been dismissed by your later commits, please double check.

**Comment by freeHackOfJeff** [comment]
Hey Qi,

Thanks for your remaining of that!  I did a mistake for the pull request via github. :)

Cheers,
-Jeff

发件人: Qi Luo <notifications@github.com<mailto:notifications@github.com>>
答复: ApolloAuto/apollo <reply@reply.github.com<mailto:reply@reply.github.com>>
日期: 2018年2月24日 星期六 下午1:57
至: ApolloAuto/apollo <apollo@noreply.github.com<mailto:apollo@noreply.github.com>>
抄送: "Liu ,Jie(BSP)" <liujie23@baidu.com<mailto:liujie23@baidu.com>>, Mention <mention@noreply.github.com<mailto:mention@noreply.github.com>>
主题: Re: [ApolloAuto/apollo] message: fix msg_received and seq_diff check-ups (#2994)


@freeHackOfJeff<https://github.com/freehackofjeff> Your changes to result2pb been dismissed by your later commits, please double check.

―

**Pull request merged**